src/cloudant/changes.py

```diff
@@ -1,112 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I went ahead and did this, but I had to split it across two commits.  Done in 	e3f598b and 4617505.